Makefile

```diff
@@ -46,7 +46,7 @@ lint: lint-flake8 lint-pylint
 
 # E501 and F401 are ignored because Pylint performs similar checks.
```

Comment:
This comment just needs to be updated for W504 since F401 is no longer ignored, per @ragabala